src/style.css

```
@@ -0,0 +1,69 @@
+* {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-bold;
+  font-style: normal;
+  font-weight: bold;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
```

```suggestion
@font-face {
  font-family: Avenir;
  font-style: normal;
  font-weight: 300;
  src: url(./fonts/Avenir-Book.ttf);
}

@font-face {
  font-family: Avenir;
  font-style: normal;
  font-weight: bold;
  src: url(./fonts/Avenir-Heavy.ttf);
}
```

No need to declare two different font-families.
You can just call it bot 'Avenir' with different font-weights and then use it in your code like:

```
  font-family: Avenir, sans-serif;
  font-weight: 300;
```
and

```
  font-family: Avenir, sans-serif;
  font-weight: bold;
```
when needed